templates/prometheus.go

```diff
@@ -31,6 +34,13 @@ var resourceSelector = metav1.LabelSelector{
 	},
 }
 
+var (
+	KubeRBACProxyPortNumber          int32  = 9339
```

Comment:
Please use `int` instead of `int32`, and convert to the specific storage class when needed
```suggestion
	KubeRBACProxyPortNumber          int  = 9339
```